core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/InvokerSupervision.scala

```diff
@@ -175,8 +175,26 @@ class InvokerPool(childFactory: (ActorRefFactory, InvokerInstanceId) => ActorRef
     val raw = new String(bytes, StandardCharsets.UTF_8)
     PingMessage.parse(raw) match {
       case Success(p: PingMessage) =>
-        self ! p
-        invokerPingFeed ! MessageFeed.Processed
+        if (p.instance.instance < 0) {
```

Comment:
if user wants to do some zerodowntime deployment, need to disable the invoker firstly and wait invoker executed all flying activations.

User can send /disable to the invoker,  in invoker backend, use the same topic name `health` to send unhealthy message via negative number of invoker.

here, get the message from kafka and make the invoker to `unhealthy` and `down` finally.